Tests/BugsnagTests/Mocks/ReporterMock.swift

```diff
@@ -19,6 +19,7 @@ internal class ReporterMock: ReporterType {
     internal func report(
         error: Error,
         request: Request?,
+        severity: Severity = .error,
```

Comment:
I don't think the mock should introduce this default behaviour. I guess it adds some convenience, but it could also lead to some unexpected behaviour.